seednode/docker/startSeedNode.sh

```diff
@@ -28,4 +54,14 @@ if [ ! -z "$USE_LOCALHOST_FOR_P2P" ]; then
     ARGS="$ARGS --useLocalhostForP2P=$USE_LOCALHOST_FOR_P2P"
 fi
 
-JAVA_OPTS='-Xms1800m -Xmx1800m' ./build/app/bin/bisq-seednode $ARGS
+while true
```

Comment:
Restarting shouldn't be builtin to the container, the docker itself can be restarted if needed

Reply:
Instead of restarting for when bitcoind has not started yet, detect whether bitcoind has started and sleep when not